kafka/datagen/datagen.go

```diff
@@ -0,0 +1,487 @@
+package datagen
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+// KafkaTopic sets the kafka topic
+var KafkaTopic = "testtopic"
+
+// PostData generates data to be used by kafka's rest proxy
+func PostData() (response map[string]interface{}, err error) {
+	krpURL := "localhost:8082"
+	postURL := fmt.Sprintf("http://%s/topics/%s", krpURL, KafkaTopic)
```

Comment:
the kafka topic should also be passed in to the function rather than being set as a global variable.

Reply:
So it may look like:
`func PostData(krpURL string, kafkaTopic string) (response map[string]interface{}, err error) {`
?

Reply:
yeah, krpURL is a very lazy name though... should probably make it `restProxyURL` or something

Reply:
Is there a way that you don't have to input those variables? Like if you wanted the default?

Reply:
Should we do it that way or force the input of the variables?

Reply:
I think this function should require that those values be passed in. The `kafkagen` program should have sensible defaults for these things (and the topic should match the one used by `pdk kafka`)